doc/ExtraDocuments/Fileformats.md

```diff
@@ -0,0 +1,248 @@
+# File formats of Magnetic field data
+
+In this section we explain the different file formats that one might encounter when dealing with magnetic field data. We state in which context they usually appear, as well as codes generating it/converting it to other formats. 
+
+For syntax: The name of each section is given by the file format (i.e. how the data is structured/arranged in the file). In paranthese one finds the type of field (2D or 3D) that is usually stored, as well as a common code that generates the files in question. Finally the file type (different from format) i.e. the `.extension` of the file (e.g. textfile, binary netcdf, ...) is listed as well.
+
+## gfiles/EQDSK format (axissymmetric 2D equillibrium, EFIT, textfile)
+
+[Original documentation from 1997](https://w3.pppl.gov/ntcc/TORAY/G_EQDSK.pdf) (note that there are some mistakes, e.g. `rcentr` does not exist) <br>
+
+Units: SI (NOTE: converters using gfiles as source may give out the quantities in cgs units after conversion) <br>
+entries per line: 5 (or less if end of points) <br>
+values format: ~ 16 characters (including sign and exponent notation) <br>
+
+While the general structure (see link above) should be the same for all files of that format, additional linebreaks and different formating of floating point numbers (number of digits) make it not possible to have one uniform reader routine for these textfiles. The user still has to choose the right reader routine for their present case, modify them accordingly or write their own.
```

Comment:
Actually, it consistently uses 16 characters for floating point data, it's just that the Fortran format contains no separating spaces. This makes it quite cumbersome to read with practically any other programming language because you have to split the string and then parse the numbers. And if Fortran can't read it, I suggest to pre-process the file instead of writing yet another reader for every sloppy output that comes our way. /rantoff

Reply:
This section was supposed to be about reading with programming languages that are not FORTRAN like PYTHON. I will make this more clear here^^